internal/app/export/export.go

```diff
@@ -78,6 +80,35 @@ func CreateExport(prefix string, runTime time.Time) (Export, error) {
 	return Export{tmpDir: tmpExportDir, fileList: []string{}, runTime: runTime}, err
 }
 
+// a different version to allow exporting to a local folder
+func CreateExportLocal(outputPath string, runTime time.Time) (Export, error) {
+	err := os.Mkdir(outputPath, dirPerm)
+	return Export{tmpDir: outputPath, fileList: []string{}, runTime: runTime}, err
+}
+
+// a different version to allow an exporter that imports files from a local folder
+func CreateExportFromLocal(inputPath string, runTime time.Time) (Export, error) {
```

Comment:
Please add tests for these functions.
The first one should be similar to TestCreateExport - just needs to ensure the path passed as argument is created.
The second one need to ensure the files contained in the path passed as argument are read correctly, something that's done within TestExport_AddFileWithDataSource.

Reply:
Hi @constantinoantunes1 - I have added TestCreateExportLocal and TestExportLocal_AddFileWithDataSource